pkg/operator/sync.go

```diff
@@ -244,20 +244,50 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 	if err != nil {
 		return err
 	}
-	// as described by the name this is essentially static, but it no worse than what was here before.  Since changes disrupt workloads
-	// and since must perfectly match what the installer creates, this is effectively frozen in time.
-	initialKubeAPIServerServingCABytes, err := optr.getCAsFromConfigMap("openshift-config", "initial-kube-apiserver-server-ca", "ca-bundle.crt")
-	if err != nil {
+
+	bootstrapComplete := false
+	_, err = optr.kubeClient.CoreV1().ConfigMaps("kube-system").Get(context.TODO(), "bootstrap", metav1.GetOptions{})
```

Comment:
I presume you want the updated information ASAP but this sync will be running a lot in-cluster so it probably is worth it to grab it from the cache? (not that we're super consistent on that)

Also, does this CM exist forever? Could it somehow be removed sometime down the line in the cluster? This `syncRenderConfig` runs first of the operator syncs and is crucial to our success so just want to make sure we don't break here

Reply:
> Could it somehow be removed sometime down the line in the cluster?

This is the in-cluster signal.  If it goes away, a lot will break.

Reply:
lister is a good idea.  Updated to a lister.